config/jobs-chromeos.yaml

```
@@ -551,3 +564,11 @@ jobs:
   tast-ui-arm64-qualcomm: *tast-ui-job
   tast-ui-x86-intel: *tast-ui-job
   tast-ui-x86-amd: *tast-ui-job
+
+  v4l2-decoder-conformance-av1:
+    <<: *v4l2-decoder-conformance-job
+    params:
+      <<: *v4l2-decoder-conformance-job-params
```

```suggestion
      <<: *v4l2-decoder-conformance-params
```

See suggestion above